features/step_definitions/common.ts

```diff
@@ -161,6 +161,24 @@ Given(
   },
 );
 
+Given(
+  "The app {string} with table field have some records as below:",
```

Comment:
@tuanphamcybozu 

I see that there is no difference between this step and the step `The app {string} has some records as below`
Could you check it again?
If It's redundant, please remove it.
Thank you.